R/infrastructure-git.R

```diff
@@ -190,10 +241,12 @@ use_git_ignore <- function(ignores, directory = ".", pkg = ".") {
 #' those fields already exist.
 #'
 #' @inheritParams use_git
+#' @param host GitHub host to use. Override with your GitHub enterprise
+#'    hostname, for example, "https://github.hostname.com".
 #' @family git infrastructure
 #' @keywords internal
 #' @export
-use_github_links <- function(pkg = ".") {
+use_github_links <- function(pkg = ".", host = "https://github.com") {
```

Comment:
I think this function should be rewritten to query the API for the repo and return the html_url and issues_url from the response (https://developer.github.com/v3/repos/#response-3) rather than special casing the api.github.com url.

The issues_url returns `https://api.github.com/repos/hadley/devtools/issues{/number}` so the `{/number}` needs to be stripped first.

```r
res <- github_GET("repos/hadley/devtools")
github_URL <- res$html_url
issues_URL <- sub("\\{/number\\}", "", res$issues_url)
```